gulpfile.babel.js

```diff
@@ -0,0 +1,115 @@
+import autoprefixer from 'gulp-autoprefixer';
+import babelify from 'babelify';
+import browserify from 'browserify';
+import concat from 'gulp-concat';
+import del from 'del';
+import domain from 'domain';
+import flatten from 'gulp-flatten';
+import filter from 'gulp-filter';
+import gulp from 'gulp';
+import gutil from 'gulp-util';
+import hbsfy from 'hbsfy';
+import plumber from 'gulp-plumber';
+import sass from 'gulp-sass';
+import tap from 'gulp-tap';
+
+
+/* Default task */
+gulp.task('default', function() {
+  gulp.start('build');
+});
+
+
+/* Removing whole ./wysadzulice/static/ directory */
+gulp.task('clean', del.bind(null, './wysadzulice/static'));
+
+
+/* Building JS */
+gulp.task('js', function() {
+  return gulp.src('./wysadzulice/assets/main.js')
+    .pipe(plumber())
+    .pipe(tap(function(file) {
+      const dom = domain.create();
+      dom.on('error', function(err) {
+        gutil.log(
+          gutil.colors.red('Browserify compile error:'),
+          err.message, '\n\t',
+          gutil.colors.cyan('in file'), file.path
+        );
+        gutil.beep();
+      });
+      dom.run(function() {
+        file.contents = browserify({
+          entries: [file.path],
+          debug: false,
+          standalone: 'Planting',
+          paths: ['./node_modules/', './wysadzulice/assets/'],
```

Comment:
You probably right. Because all js modules available in `wysadzulice/assets` will be imported into main using relative paths, it looks, that we can freely remove this option from browserify.

I will do it at today's meeting.